### PR TITLE
initial 1.2.0 ver

### PR DESCRIPTION
**Pull request by M-Waszkiewicz-Anaconda**
jinxed 1.2.0, dependency for blessed >=1.19.0
- [upstream](https://github.com/Rockhopper-Technologies/jinxed/tree/1.2.0)
- [pypi](https://pypi.org/project/jinxed/)
- [CF](https://github.com/conda-forge/jinxed-feedstock)
- updated recipe to satisfy linter
- added tests
- not fully on par with ncurses >= 6.3, therefore some tests skipped

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,34 +12,47 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  Skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
```
Comment (boldorider4) [inline recipe/meta.yaml:16]: `--no-build-isolation`, the second time I see this today...did we decide to make it part of the run-of-the-mill build command?

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml:16]: it is part on the linter requriement, actually it was generated by the linter --fix command, is there anything I missing here ?

Comment (cbouss) [reply recipe/meta.yaml:16]: It was decided more than 6 months ago. On this package that doesn't change much, but on packages with build depdencies set in pyproject.toml, that prevents pip from building in isolated mode, disregarding our deps and fetching from pypi.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,34 +12,47 @@ source:
...
 
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python
+    - wheel
```
Comment (cbouss) [inline recipe/meta.yaml:22]:
```suggestion
    - wheel
    - setuptools
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,34 +12,47 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  Skip: True  # [py<38]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
it declares support for older versions: https://github.com/Rockhopper-Technologies/jinxed/blob/1.2.0/setup.py#L54

File: recipe/meta.yaml [hunk 4]
```diff
@@ -12,34 +12,47 @@ source:
...
 about:
   home: https://pypi.org/project/jinxed/
   summary: Jinxed Terminal Library
+  description: | 
+    Jinxed is an implementation of a subset of the Python curses library. It provides pure Python implementations of 
+    terminfo functions such as tigetstr() and tparm() as well as convenience methods for working with Windows terminals.
+  license_family: OTHER
   license: MPL-2.0
   license_file: LICENSE
+  doc_url: https://jinxed.readthedocs.io/en/stable/
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://jinxed.readthedocs.io
```

File: recipe/meta.yaml [hunk 5]
```diff
@@ -12,34 +12,47 @@ source:
...
   commands:
     - pip check
+    - pytest -vv tests  # [py<311]
+    # seems from the tests that jinxed 1.2.0 does not support well compability tests 
+    # with ncurses >= 6.3, ncurses < 6.3 is not available for py>=311
+    - pytest -k "not test_tparm.py" -vv  # [py>=311 or not win]
   requires:
     - pip
+    - pytest
+    - ncurses <6.3 # [py<311 or not win]
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
```
All our python are built with `- ncurses >=6.4,<7.0a0`. By setting <6.3 you are restricting to old builds of 3.10, 3.9, 3.8. Skipping the tests with a comment is fine. You can open an issue upstream too.

Comment (M-Waszkiewicz-Anaconda) [reply recipe/meta.yaml]: raised issue in upstream https://github.com/Rockhopper-Technologies/jinxed/issues/4

**Review by skupr-anaconda — APPROVED** [review]

**Review by danpetry — APPROVED** [review]

**Pull request merged**